hal/armv7m/imxrt/10xx/config.h

```diff
@@ -17,20 +17,24 @@
 #ifndef _CONFIG_H_
 #define _CONFIG_H_
 
+#ifndef __ASSEMBLY__
+
 #include "imxrt.h"
 #include "peripherals.h"
 #include "../types.h"
 #include "../../mpu.h"
 #include "../../string.h"
 
-
 /* User interface */
 #define PATH_KERNEL "phoenix-armv7m7-imxrt106x.elf"
 
-/* Addresses descriptions */
-#define ADDR_SYSPAGE 0x20200000
-#define SIZE_STACK   5 * 1024
+#endif
+
 #define SIZE_PAGE    0x200
+#define SIZE_STACK   4 * SIZE_PAGE
+#define SIZE_SYSPAGE 8 * SIZE_PAGE
+
+#define ADDR_STACK _end + SIZE_SYSPAGE + SIZE_STACK
```

Comment:
Put compound defines in brackets
```suggestion
#define SIZE_STACK   (4 * SIZE_PAGE)
#define SIZE_SYSPAGE (8 * SIZE_PAGE)

#define ADDR_STACK (_end + SIZE_SYSPAGE + SIZE_STACK)
```